microsoft/testsuites/lis/lissuite.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy import assert_that
+
+from lisa import Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.tools import Wget
+from lisa.util import SkippedException
+
+
+@TestSuiteMetadata(
+    area="lis",
+    category="functional",
+    description="""
+    ##################################
+    """,
+)
+class lis(TestSuite):  # noqa
+    @TestCaseMetadata(
+        description="""
+            #################
+        """,
+        priority=1,
+    )
+    def verify_lis_driver_version(self, node: Node) -> None:
+        wget_tool = node.tools[Wget]
+        rpm_status = node.execute(
+            'rpm -qa | grep "kmod-microsoft-hyper-v" && rpm -qa '
+            '| grep "microsoft-hyper-v"'
+        ).exit_code
+
+        if rpm_status != 0:
+            raise SkippedException("No LIS RPM's are detected. Skipping test.")
+        version = node.execute(
+            """modinfo "hv_vmbus" | grep version: | head -1 | awk '{print $2}'"""
+        ).stdout
+
+        for i in range(5, 8):
+            node.execute("rm -rf hv_compat.h")
+            wget_tool.get(
+                f'https://raw.githubusercontent.com/LIS/lis-next/"'
+                f'{version}"/hv-rhel{i}.x/hv/include/linux/hv_compat.h',
+                filename="hv_compat.h",
+                file_path="./",
+            )
+            source_version = node.execute(
+                """grep 'define HV_DRV_VERSION' hv_compat.h|cut -d '"' -f 2"""
+            ).stdout
+            source_version_hex = node.execute(
+                "grep 'define _HV_DRV_VERSION' hv_compat.h|cut -d ' ' -f 3|tr -d '0x'"
+            ).stdout
+
+            check_res = self._check_lis_version(node, version, source_version)
+            check_hex_res = self._check_lis_version_hex(
+                node, version, source_version_hex
+            )
+
+            assert_that(check_res and check_hex_res).described_as(
+                f"The result of _check_lis_version() was {check_res}. "
+                f"The result of _check_lis_version_hex() was {check_hex_res}."
+            ).is_equal_to(True)
+        pass
+
+    # returns true if version and source_version are the same
+    def _check_lis_version(self, node: Node, version: str, source_version: str) -> bool:
+        self.__log.debug("Detected modinfo version is {version}")
```

Comment:
Don't use the `self.__log`. It's for test suite. Use the log from test method, like other test cases.